### DISCUSSION TITLE
Performance time in terminal

### DISCUSSION THREAD

**Discussion by improcode** [vercel/next.js#65074] [Help]
### Summary

I can't find exact information on what the terminal clock measures and in what unit, please help
![Zrzut ekranu 2024-04-26 o 09 46 45](https://github.com/vercel/next.js/assets/37656895/39d2e858-f06b-44cb-b551-f8d2db6fd903)

**Comment by icyJoseph** [comment]
Hi,

Next.js logs to the terminal usually contain a lot more info and are properly formatted.

This is likely coming from a library you are using. It won't be easy to tell...
